tests/test_generator.py

```diff
@@ -312,159 +355,160 @@ def test_write_tagged_file(fileutils, generator_instance: Generator):
         "1200": "120",
         "VALID": "valid",
     }
-    cmd_list = generator_instance._write_tagged_files(
-        files=files, params=param_set, dest=generator_instance.root
-    )
+    to_configure = [
+        ConfigureOperation(src=file, dest=dest, file_parameters=file_param)
+        for file in source
+    ]
+    gen = GenerationContext(pathlib.Path(test_dir))
+    cmd_list = generator_instance._configure_files(files=to_configure, context=gen)
     assert isinstance(cmd_list, CommandList)
-    for file, cmd in zip(tagged_files, cmd_list):
-        assert file in cmd.command
+    # Extract file paths from commands
+    cmd_src_paths = set()
+    for cmd in cmd_list.commands:
+        # Assert destination path exists in cmd
+        assert create_final_dest(pathlib.Path(test_dir), dest) in cmd.command
+        src_index = cmd.command.index("configure") + 1
+        cmd_src_paths.add(cmd.command[src_index])
+    # Assert all file paths are in the command list
+    file_paths = {str(file) for file in source}
+    assert file_paths.issubset(
+        cmd_src_paths
+    ), "Not all file paths are in the command list"
 
 
-def test_write_tagged_directory(fileutils, generator_instance: Generator):
-    """Test Generator._write_tagged_files helper function with directory path"""
-    config = get_gen_file(fileutils, "tag_dir_template")
-    files = EntityFiles(tagged=[config])
-    param_set = {"PARAM0": "param_value_1", "PARAM1": "param_value_2"}
-    cmd_list = generator_instance._write_tagged_files(
-        files=files, params=param_set, dest=generator_instance.root
-    )
+# TODO Add configure_file tests
```

Comment:
So I was thinking that since test_file_operations.py tests this, that I did not need to in this file but please let me know otherwise! I was thinking that testing _execute_commands calls the correct entry points was sufficient

Reply:
^^ Totally makes sense to me! In that case can this TODO be removed entirely?